### DISCUSSION TITLE
ctrl + arrow keys move slider control points

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#23833] [Ideas]
When you select a hitcircle, or slider, using ctrl + arrow keys will move it editor units to a direction. this is useful for small adjustments. it would be nice to have this ability for control points, and if you selected a control point in the manner demonstrated in the attached image, then using ctrl + arrow keys would move just that control point, as currently it moves them all and thus the entire slider.

![image](https://github.com/ppy/osu/assets/78441708/aa3dccfb-3163-4b7e-80cb-ea31d24e6bc4)

**Comment by peppy** [comment]
I took a look at what this would require implementation wise and while it's not too hard, requires some refactoring. The current logic handles a drag start and end, which would need to be similarly handled for key down/up.
